config.py

```diff
@@ -121,6 +125,9 @@ class TestingConfig(DevelopmentConfig):
     REQUESTS_GET_TIMEOUT = 99
     REQUESTS_POST_TIMEOUT = 99
     DATABASE_SCHEMA = 'partysvc'
+    DB_POOL_SIZE = 5
+    DB_MAX_OVERFLOW = 10
+    DB_POOL_RECYCLE = -1
```

Comment:
We don't need these in the Testing config as they'll use the default values in the Config class